src/runtime/mod.rs

```diff
@@ -80,6 +86,10 @@ pub struct Module {
     pub entrypoint: Option<String>,
     /// The name of the host.
     pub host: Option<String>,
+    /// The URL fragment for the bindle server.
+    ///
+    /// If none is supplied, then http://localhost:8080/v1 is used
+    pub bindle_server: Option<String>,
```

Comment:
Nit: This should probably just be instantiated with the default at creation time rather than allocating a new `String` each time we check for the bindle_server and find `None`